ms365_accountcreator/templates/index.html

```diff
@@ -73,20 +73,50 @@ <h1>{{ gettext('Account Registration for Microsoft Teams for University of Stutt
     </div>
 
     <script>
-        function emailSubmit(event) {
-            event.preventDefault();
+        const url_api_endpoint_email_verification = "{{url_api_endpoint_email_verification}}"
+        const url_api_endpoint_account_creation = "{{url_api_endpoint_account_creation}}"
 
-            document.getElementById('step-1').classList.add('done');
+        function postData(url = '', data = {}) {
+            return fetch(url, {
+                method: 'POST',
+                mode: 'cors',
+                cache: 'no-cache',
+                headers: {
+                    'Content-Type': 'application/json'
+                },
+                body: JSON.stringify(data)
+            });
+        }
 
-            const step2 = document.getElementById('step-2');
-            step2.classList.remove('inactive');
-            step2.setAttribute('open', '');
+        function emailSubmit(event) {
+            event.preventDefault();
+            if(! document.getElementById('input-consent-mail').checked) {
+                alert("{{ gettext('Consent needed') }}")
+                return false;
+            }
 
+            let email = document.getElementById('input-email').value
+            postData(url_api_endpoint_email_verification, {'email': email}).then(response => {response.json().then(
```

Comment:
```suggestion
            postData(url_api_endpoint_email_verification, {'email': email})
                .then(response => response.json())
                .then(response_data =>
```
Use promise chaining instead of promise nesting. (the suggestion needs to be testet)